ui/00_message/ui/system_log.xml

```diff
@@ -141,43 +141,43 @@
   </message>
   <message>
     <key>syslog_pawn_rescue_start</key>
-    <text xml:space="preserve">&lt;NAME PAWN&gt;を救出してください</text>
+    <text xml:space="preserve">&lt;NAME PAWN&gt;'s rescue mission has begun.</text>
```

Comment:
Not sure of the context here but the kana says "Please rescue (NAME PAWN)". I might be wrong.